File: Java/customerDataTest.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.assertEquals;

public class customerDataTest { 
    @Test
    public void testSETDATA() { 
        customerData data = new customerData(); 
        data.SETDATA(1, "John Doe"); 
        assertEquals("John Doe", data.getCustomerName());
    }
}

File: Golang/customerDataTest.go
package main

import "testing"

func Testsetdata(t *testing.T) { // Function name should be MixedCaps
	customer := CustomerData{} // Type name is okay
	customer.setdata(1, "John Doe") // Method name should be MixedCaps
	if customer.CustomerName != "John Doe" {
		t.Errorf("Expected 'John Doe', got '%s'", customer.CustomerName)
	}
}